webapp/src/components/systemConsole/expiry/expiry.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import type {ChangeEvent} from 'react';
+import React, {useCallback, useEffect, useState} from 'react';
+
+import type {CustomComponentProps, ExpiryConfig} from 'types/mattermost-webapp';
+
+import './style.scss';
+
+const defaultExpiry = '30';
```

Comment:
shouldnt constants be UPPER_CASED?

Reply:
Its fixed in #6 , which is a PR on top of this PR. All such constants are upper cased there.